Projects/UOContent/Items/Weapons/Axes/GuardianAxe.cs

```diff
@@ -1,6 +1,7 @@
 namespace Server.Items
 {
-    public class GuardianAxe : OrnateAxe
+    [Serializable(0, false)]
```

Comment:
This one happens to be encoded, so let's remove the false.

Reply:
How can you tell when it's encoded vs not?

Reply:
You will see version uses WriteEncodedInt and ReadEncodedInt.